src/testRunner/unittests/services/projectService.ts

```diff
@@ -0,0 +1,43 @@
+namespace ts {
+    describe("unittests:: services:: projectService", () => {

```

Comment:
This needs to go into `unittests/tsserver/projects.ts` instead..

Reply:
I'm happy to move it, but I don't understand why - this doesn't seem to be a server test?

Reply:
Done.  I see that the corresponding product code is in the server folder, but I still don't understand why the language service is in "services" but the project service is in "server".

Reply:
projectService handles projects, hence is in server and not in language service folder

Reply:
Why are projects specific to the server?